src/main/java/io/github/org/programming/backend/handler/SlashCommandHandler.java

```diff
@@ -110,22 +112,27 @@ private void onFinishedRegistration() {
     public void onSlashCommandInteraction(@NotNull SlashCommandInteractionEvent event) {
         final SlashCommandExtender cmd = slashCommand.get(event.getName());
 
-        if (cmd.isOwnerOnly() && event.getUser().getIdLong() != botOwnerId()) {
+        if (cmd.build().isOwnerOnly() && event.getUser().getIdLong() != botOwnerId()) {
```

Comment:
I don't understand what is `build` for

Reply:
Have a look at the example command. It i used to build the command, like name, description, adding options ,etc